src/lib/utils/meta_tables/meta_system_utilization_table.cpp

```diff
@@ -0,0 +1,337 @@
+#ifdef __linux__
+
+#include <numa.h>
+#include <fstream>
+
+#endif
+
+#ifdef __APPLE__
+
+#include <mach/mach.h>
+
+#endif

```

Comment:
See above. Also, we could reduce the number of empty lines here; they don't really improve the readability.